src/redux/index.js

```diff
@@ -0,0 +1,22 @@
+import { combineReducers } from 'redux';
+import { idReducer } from './reducers/id';
+import { nameReducer } from './reducers/name';
+import { fatherNameReducer } from './reducers/fatherName';
+import { ageReducer } from './reducers/age';
+import { addressReducer } from './reducers/address';
+import { dataReducer } from './reducers/data';
+import { editButtonReducer } from './reducers/editbutton';
+import { createStore } from 'redux';
+
+const rootReducer = combineReducers({
+  id: idReducer,
+  name: nameReducer,
+  fatherName: fatherNameReducer,
+  age: ageReducer,
+  address: addressReducer,
+  data: dataReducer,
+  editButton: editButtonReducer,
```

Comment:
name your reducer exactly the same as your state variable so that you can write {id} instead of {id:idreducer}